server/.env.example

```
@@ -0,0 +1,2 @@
+# Server locale: "en" or "fr" (see locales/*.ftl and src/l10n.rs). Unset or invalid → English.
+DEFAULT_LANGUAGE = "en"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)